notimetowait2.md

```diff
@@ -60,11 +60,11 @@ REGISTRATION IS NOW OPEN
 
 [http://bit.ly/nttw2signup](http://bit.ly/nttw2signup)
 
-## Call for Proposals:
+## Program:
 
-CALL FOR PROPOSALS IS OPEN UNTIL SEPTEMBER 22, 2017.
+Below is the preliminary program.  Please note that sessions are still being finalized and program times, titles, and speakers are subject to change. The program will be updated again by October 5th. Questions or comments about the program may be set to [info@mediaarea.net](mailto:info@mediaarea.net) or provided as a comment within the Google spreadsheet.
```

Comment:
(double space after first period)

I would remove "The program will be updated again by October 5th." as this is already said "preliminary", and program may change also after Oct 5.

Reply:
+1